tests/check/check_h26xsigned_sign.c

```diff
@@ -714,7 +714,7 @@ signed_video_suite(void)
 #endif
   // tcase_add_loop_test(tc, correct_nalu_sequence_with_eos, s, e);
   // tcase_add_loop_test(tc, correct_multislice_sequence_with_eos, s, e);
-  tcase_add_loop_test(tc, correct_nalu_sequence_without_eos, s, e);
+  tcase_add_loop_test(tc, correct_nalu_sequence_without_eos, s, 1);
```

Comment:
You need to temporarily change the meson as well. Otherwise the threaded plugin will not be executed in the tests and memory leaks cannot be detected.